### ISSUE TITLE
trueLightning == "false" case has the same effect as "true"

### ISSUE THREAD

**Issue by Tagir-1** [Gameslinx/Tessellation#45]
Hi, I have no idea what all these methods in ParallaxSource.cs do, but your logic for trueLightning seems to contain error. You don't seem to catch trueLighting = "false" case properly. The case when trueLighting is false and the case when it is adaptive but without atmosphere both lead to the same last option, which is duplicate of the case when trueLighting is true. This makes both "true" and "false" in config produce the same pitch black textures at night.
You should have something like this:

```c
//True lighting is on everywhere, or adaptive is on and planet is without atmosphere (makes it dark at night)
if (ParallaxSettings.trueLighting == "true" || (FlightGlobals.GetBodyByName(planetName).atmosphere == false && ParallaxSettings.trueLighting == "adaptive"))
            {
                parallaxMaterial.EnableKeyword("ATTENUATIONOVERRIDE_ON");
                parallaxMaterial.DisableKeyword("ATTENUATIONOVERRIDE_OFF");
                //etc...
            }
            else
            {
                //True lighting if off, or it is adaptive in atmosphere (ambient light at night)
                if ((ParallaxSettings.trueLighting != "true" && ParallaxSettings.trueLighting != "adaptive") || (FlightGlobals.GetBodyByName(planetName).atmosphere == true && ParallaxSettings.trueLighting == "adaptive"))
                {
                    parallaxMaterial.EnableKeyword("ATTENUATIONOVERRIDE_OFF");
                    parallaxMaterial.DisableKeyword("ATTENUATIONOVERRIDE_ON");
                    //etc...
                }
            }
```
Also, i've noticed that 1.2.1 release contains older version of ParallaxSource.cs

**Comment by Gameslinx** [comment]
Thank you. Just released 1.2.2 in time to see this. 1.2.3 will address these issues. Thank you for the report

**Comment by Gameslinx** [comment]
Corrected - Will be fixed in 1.2.3!
